docs/P11.md

```diff
@@ -0,0 +1,134 @@
+# P11 Kit Configuration
+
+Below, will be examples and discussion on how to use P11kit with tpm2-pkcs11. This will not be
+exhaustive, nor should it be.
```

Comment:
However, commands not explicitly listed will likely not work as they are work in progress.